clar2wasm/src/words/equal.rs

```diff
@@ -1264,4 +1253,50 @@ mod tests {
         (is-eq (var-get a) (var-get b))";
         crosscheck(snippet, Ok(Some(clarity::vm::Value::Bool(true))));
     }
+
+    //
+    // Module with tests that should only be executed
+    // when running Clarity::V2 or Clarity::v3.
+    //
+    #[cfg(not(feature = "test-clarity-v1"))]
+    #[cfg(test)]
+    mod clarity_v2_v3 {
+        use super::*;
+        use crate::tools::crosscheck;
+
+        #[test]
+        // TODO: see issue #496.
+        // The test below should pass when running it in ClarityV1.
+        // It should be removed from this module when the issue is fixed.
+        #[cfg(not(feature = "test-clarity-v1"))]
+        fn index_of_complex_type() {
+            crosscheck(
+                "(index-of (list (list (ok 2) (err 5)) (list (ok 42)) (list (err 7))) (list (err 7)))",
+                Ok(Some(Value::some(Value::UInt(2)).unwrap())),
+            );
+        }
```

Comment:
nit: redundant `#[cfg(not(feature = "test-clarity-v1"))]`

Reply:
Good catch. Removed.